openpype/hosts/houdini/plugins/create/create_pointcache.py

```diff
@@ -1,48 +1,45 @@
+# -*- coding: utf-8 -*-
 from openpype.hosts.houdini.api import plugin
+from openpype.pipeline import CreatedInstance
 
+import hou
 
-class CreatePointCache(plugin.Creator):
-    """Alembic ROP to pointcache"""
 
-    name = "pointcache"
+class CreatePointCache(plugin.HoudiniCreator):
+    """Alembic ROP to pointcache"""
+    identifier = "pointcache"
     label = "Point Cache"
     family = "pointcache"
     icon = "gears"
 
-    def __init__(self, *args, **kwargs):
-        super(CreatePointCache, self).__init__(*args, **kwargs)
-
-        # Remove the active, we are checking the bypass flag of the nodes
-        self.data.pop("active", None)
-
-        self.data.update({"node_type": "alembic"})
-
-    def _process(self, instance):
-        """Creator main entry point.
+    def create(self, subset_name, instance_data, pre_create_data):
+        instance_data.pop("active", None)
+        instance_data.update({"node_type": "alembic"})
 
-        Args:
-            instance (hou.Node): Created Houdini instance.
+        instance = super(CreatePointCache, self).create(
+            subset_name,
+            instance_data,
+            pre_create_data)  # type: CreatedInstance
 
-        """
+        instance_node = hou.node(instance.get("members")[0])
         parms = {
-            "use_sop_path": True,  # Export single node from SOP Path
-            "build_from_path": True,  # Direct path of primitive in output
-            "path_attrib": "path",  # Pass path attribute for output
+            "use_sop_path": True,
+            "build_from_path": True,
+            "path_attrib": "path",
             "prim_to_detail_pattern": "cbId",
-            "format": 2,  # Set format to Ogawa
-            "facesets": 0,  # No face sets (by default exclude them)
-            "filename": "$HIP/pyblish/%s.abc" % self.name,
+            "format": 2,
+            "facesets": 0,
+            "filename": "$HIP/pyblish/{}.abc".format(self.identifier)
```

Comment:
This shouldn't be the identifier but the unique subset's name so that two Pointcache instances don't write to the same `pointcache.abc` file, right?